src/figuresClasses.ts

```diff
@@ -1,19 +1,80 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
```suggestion
  shape: string;
  color: string;
```
Use enums for color and shape